aci/resource_aci_vmmusraccp.go

```diff
@@ -0,0 +1,203 @@
+package aci
+
+import (
+	"fmt"
+
+	"github.com/ciscoecosystem/aci-go-client/client"
+	"github.com/ciscoecosystem/aci-go-client/models"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func resourceAciVMMCredential() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAciVMMCredentialCreate,
+		Update: resourceAciVMMCredentialUpdate,
+		Read:   resourceAciVMMCredentialRead,
+		Delete: resourceAciVMMCredentialDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: resourceAciVMMCredentialImport,
+		},
+
+		SchemaVersion: 1,
+
+		Schema: AppendBaseAttrSchema(map[string]*schema.Schema{
+			"vmm_domain_dn": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"annotation": &schema.Schema{
+				Type: schema.TypeString,
+
+				Optional:    true,
+				Computed:    true,
+				Description: "Mo doc not defined in techpub!!!",
+			},
```

Comment:
Annotation field is not needed here, because I added annotation in the base attribute schema.